.github/workflows/deploy-stacks.yml

```diff
@@ -0,0 +1,117 @@
+name: deploy-stacks.yml
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  create-env-change-sets-and-deploy:
+    permissions:
+      id-token: write
+      contents: read
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node and install cdk cli
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Install dependencies and build project
+        run: |
+          npm install
+          npm install -g aws-cdk
+          npm run build
+
+      - name: Set up Env variables
+        run: |
+          echo "BETA_ACCOUNT_ID=${{ secrets.BETA_ACCOUNT_ID }}" >> $GITHUB_ENV
+          echo "PROD_ACCOUNT_ID=${{ secrets.PROD_ACCOUNT_ID }}" >> $GITHUB_ENV
+          echo "ASSUMED_ROLES=${{ secrets.ASSUMED_ROLES }}" >> $GITHUB_ENV
+          echo "REGION=${{ secrets.REGION }}" >> $GITHUB_ENV
```

Comment:
REGION does not need to be a secret right? Nothing to hide in there.

Reply:
I'm okay if you want to remove it.

Reply:
Yes lets do that. Secrets are not visible once added in GH.